crates/rgb-frame/src/lib.rs

```diff
@@ -154,22 +154,22 @@ impl Frame {
 #[repr(u32)]
 #[strum(serialize_all = "UPPERCASE")]
 pub enum FourCC {
-    /// x:R:G:B [31:0] little endian
+    /// x:R:G:B `\[` 31:0 `\]` little endian
```

Comment:
What do these changes do?

Reply:
Escape the slashes. `cargo doc` rendered these with warnings.